drivers/include/tps6274x.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2017 RWTH Aachen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_tps6274x TPS6274x
+ * @ingroup     drivers_power
+ * @brief       Device driver interface for the TPS6274x DC-DC Converter
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the TPS6274x DC-DC Converter
+ *
+ * @author      Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author	Josua Arndt <jarndt@ias.rwth-aachen.de>
+ */
+
+#ifndef TPS6274X_H
+#define TPS6274X_H
+
+#include <inttypes.h>
+#include "periph/gpio.h"
+
+/**
+ * @name Configuration struct
+ * @{
+ */
+typedef struct {
+    gpio_t vsel1_pin;   /** vsel1 pin mapping */
+    gpio_t vsel2_pin;   /** vsel2 pin mapping */
+    gpio_t vsel3_pin;   /** vsel3 pin mapping */
+    gpio_t vsel4_pin;   /** vsel4 pin mapping */
+    gpio_t ctrl_pin;    /** ctrl pin  mapping */
+}tps6274x_config_t;
+/** @} */
+
+
+/**
+ * @brief init converter
+ *
+ * @param[in] voltage		Voltage to set in mV (needs to be between 1.8V-3.3V
+ * @return			set voltage in mV
+ *
+ */
+
+uint16_t tps6274x_init(uint16_t voltage);
```

Comment:
why uint16_t? In RIOT, you can assume that "unsigned" is at least 16bit wide. Might make code smaller on 32bit platforms if using a word-sized variable...